src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,20 @@
-const fizzBuzz = number => {};
+const fizzBuzz = (number) => {
```

Comment:
You can omit the brackets here as its a single argument:
```suggestion
const fizzBuzz = number => {
```